src/nfts/NftsLoadError.test.tsx

```diff
@@ -1,10 +1,18 @@
 import { fireEvent, render } from '@testing-library/react-native'
 import React from 'react'
+import { NftEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import NftLoadError from 'src/nfts/NftsLoadError'
 
+jest.mock('src/analytics/ValoraAnalytics')
```

Comment:
nit: this can be removed ValoraAnalytics is already [mocked by default](https://github.com/valora-inc/wallet/blob/5a982019f89ef552c8fc9e5f1b0fee7bdedd7c19/__mocks__/src/analytics/ValoraAnalytics.ts).